src/profile/profilePane.utils.ts

```diff
@@ -0,0 +1,10 @@
+import { IndexedFormula, Node } from 'rdflib'
+import { Namespaces } from 'solid-namespace'
+
+export function getLabel (subject: Node, kb: IndexedFormula, ns: Namespaces) {
+  var types = kb.findTypeURIs(subject)
+  if (types[ns.foaf('Person').uri] || types[ns.vcard('Individual').uri]) {
+    return 'Your Profile'
+  }
```

Comment:
I don't understand the logic here.  If this is a view, then the label will  be non-null if the view applies to the subject.  This condition means that the subject is a person.  The view will be of that person't profile, not the user's profile. 

If it is now a view but a applet, -- view/edit my profile -- then the subject is irrelevant. It should alway s be available, no matter what the user is looking at.

Reply:
This logic is nevertheless outdated, as it was only used in editProfile, and we don't need to label it as it is only used by the dashboard anyway.

I can't recall the exact reasoning for the logic in the code above, but perhaps @Vinnl does? (I'll ask him to comment on Monday.)

Reply:
I don't know either, [it was added by Tim two years ago](https://github.com/solid/solid-panes/commit/e3d160b31ee4044afd9a080a2f896a1fb516abfa#diff-c6baf3719ead006cc587c2d4d7368fa6R22-R25).

Reply:
That was logic for a view (pane), not an app.    Trusted applications is an app.  Hence the nonsense. And the label then was "friends" -- which could apply to anyone when looking at someone else's profile, not "your profile", which does not make sense when looking at someone else.   But now I see more or less the route by which we got here.

Reply:
One reason why we find it difficult to look at this is that it has been split off as too small a module.  The entire code of the module is 
```
export function getLabel (subject: Node, kb: IndexedFormula, ns: Namespaces) {
  var types = kb.findTypeURIs(subject)
  if (types[ns.foaf('Person').uri] || types[ns.vcard('Individual').uri]) {
    return 'Your Profile'
  }
  return 'Edit your profile' // At the moment, just allow on any object. Like home pane
}
```
The name of the function, getLabel doesn't describe what it does, which is something like "return a pane label iff the subject is a person".   Which is what we do not want.  When you split off  functionality into  another module, it is really important that the reader of the calling module  understands just what is going on in the call.   This is in fact a logical condition for the app to appear, and at the top level of the app is a good place for it.  Its a top level thing.  Plus, there is a cognitive load and a quantum of effort required when the reader switches to the called module to see  what is  going on.  Don't waste that effort for two lines, keep it for 20 or more lines.   I suggest
- reintegrate those two lines into the calling code
- remove the test on the subject.'s class as it is irrelevant to an app pane
- just return a constant string as a label 'profile' say.    (module I18n later)